src/test/java/api/requests/RequestsAPILoanRenewalTests.java

```diff
@@ -67,6 +73,80 @@ public void allowRenewalLoanByBarcodeWhenFirstRequestInQueueIsHold() throws Inte
     assertThat(response.getJson().getString("action"), is("renewed"));
   }
 
+  @Test
+  public void forbidRenewalLoanByBarcodeWhenProfileIsRollingFirstRequestInQueueIsHoldAndRenewingIsDisallowedInLoanPolicy()
+    throws InterruptedException, MalformedURLException, TimeoutException, ExecutionException {
+
+    final InventoryItemResource smallAngryPlanet = itemsFixture.basedUponSmallAngryPlanet();
+    final IndividualResource rebecca = usersFixture.rebecca();
+
+    loanPolicyWithRollingProfileAndRenewingIsForbiddenWhenHoldIsPending();
+
+    loansFixture.checkOutByBarcode(smallAngryPlanet, rebecca);
+    requestsFixture.place(new RequestBuilder()
+      .hold()
+      .forItem(smallAngryPlanet)
+      .withPickupServicePointId(servicePointsFixture.cd1().getId())
+      .by(usersFixture.charlotte()));
+
+    Response response = loansFixture.attemptRenewalById(smallAngryPlanet, rebecca);
+
+    String message = response.getJson().getJsonArray("errors").getJsonObject(0).getString("message");
+    assertThat(message, is(LoanPolicy.CAN_NOT_RENEW_ITEM_ERROR));
+  }
+
+
+  @Test
+  public void allowRenewalLoanByBarcodeWhenProfileIsFixedFirstRequestInQueueIsHoldAndRenewingIsAllowedInLoanPolicy()
```

Comment:
Could you please add an assert to verify that **alternate renewal period for loans with pending hold request** is applied?

Reply:
@dmytrotur Done.